src/client/ClientDataResolver.js

```diff
@@ -244,7 +244,7 @@ class ClientDataResolver {
         if (/^https?:\/\//.test(resource)) {
           request.get(resource)
             .set('Content-Type', 'blob')
-            .end((err, res) => err ? reject(err) : resolve(res.body));
+            .end((err, res) => err ? reject(err) : resolve(res.body instanceof Buffer ? res.body : null));
```

Comment:
My testing so far has shown that returning null will be handled as a rejection by any other layer, but I'll make it an explicit reject if you want. 👍